src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -595,9 +615,22 @@ private void replaceEC2Servers() {
                     // different instance type and/or ami exists for graph building, so update the number of instances
                     // to start up to be the full amount of instances.
                     status.numServersRemaining = Math.max(otpServer.ec2Info.instanceCount, 1);
-                    if (recreateBuildImageExecutor == null) {
-                        // no recreate build image job was created, terminate the graph building instance immediately
-                        ServerController.terminateInstances(ec2, graphBuildingInstances);
+                    if (!otpServer.ec2Info.recreateBuildImage) {
+                        // the build image should not be recreated, so immediately terminate the graph building
+                        // instance. If image recreation is enabled, the graph building instance is terminated at the
+                        // end of the RecreateBuildImageJob.
+                        try {
+                            ec2.terminateInstances(
```

Comment:
Why is this no longer using `ServerController.terminateInstances` (which is used elsewhere)?